iFixit/react-commerce
# Product Page: Add Google Analytics

Tracks product views and add to cart events in Google Analytics.

## QA

See QA steps from https://github.com/iFixit/react-commerce/pull/594. The goal is feature parity with product page tracking on iFixit/ifixit.

Closes #800
